README.md

```diff
@@ -6,7 +6,7 @@
 [![Build Status][travis-image]][travis-url]
 [![Test Coverage][coveralls-image]][coveralls-url]
 
-Parse http requests with content-type `multipart/form-data`, also known as file uploads.
+Parse http requests with content-type `multipart/form-data`, also known as file uploads, `multipart/mixed` or `multipart/related`.
```

Comment:
Can you please not add multipart/related? As I referenced, we don't correctly support it and will remove this type.

Reply:
Ok